src/main/java/com/spikes2212/command/genericsubsystem/smartmotorcontrollersubsystem/SparkMaxGenericSubsystem.java

```diff
@@ -106,7 +111,8 @@ public void pidSet(UnifiedControlMode controlMode, double setpoint, PIDSettings
                        FeedForwardSettings feedForwardSettings, TrapezoidProfileSettings trapezoidProfileSettings) {
         configPIDF(pidSettings, feedForwardSettings);
         configureTrapezoid(trapezoidProfileSettings);
-        master.getPIDController().setReference(setpoint, controlMode.getSparkMaxControlType());
+        master.getPIDController().setReference(setpoint, controlMode.getSparkMaxControlType(),
+                PID_SLOT, feedForwardSettings.getkS() * Math.signum(setpoint) + feedForwardSettings.getkG());
```

Comment:
what if my position is `-600` and my setpoint is `-300`, would the `Math.signum` work then?